stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/Extensions.kt

```diff
@@ -115,3 +116,7 @@ const val CHANNEL_ARG_DRAFT = "draft"
 
 val Channel.isDraft: Boolean
     get() = getExtraValue(CHANNEL_ARG_DRAFT, false)
+
```

Comment:
small extra new line here

Reply:
🤔  Should we add docs even for code outside the SDK? I think it would be too much documentation just for the sample app... In the end, the code of the sample is small and easy to follow.